.github/workflows/ci-js.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     env:
       REPORT_DIR: reports
     name: Testing and upload coverage
-    runs-on: 
+    runs-on:
       - self-hosted
-      - gsa_runner
+      - gsa-runner
```

Comment:
Unfortunately the runners label is `gsa_runner` not `gsa-runner`.

Reply:
this repo runner will be removed. so i think we should remove the whole line